slide.sh

```diff
@@ -1,13 +1,22 @@
 #!/bin/bash
+
+let CURRENTPAGE=0
+let TOTALSLIDES=$(grep "<<EOF" slideshow.sh | wc -l | tr -d " ")
 function slide() {
+    clear
     local -r TPUT=$(type -p tput)
     [ -x "$TPUT" ] || exit 1
-    local -r IFS='' MESSAGE=${1:-<Enter> Next slide | <ctrl+c> Quit}
     local -r COLORS=(red=31 green=32 yellow=33 blue=34 purple=35 cyan=36 end=)
     local -ri COLS=$($TPUT cols) ROWS=$($TPUT lines)
     local -i CENTER=0 LINENUM=0 CTRPOS=0 MSGPOS=0 HASCOLOR=1
     local LINE='' BARE=''
     trap "$TPUT clear" 0
+    let CURRENTPAGE++
+    printf " Loading slide [$CURRENTPAGE/$TOTALSLIDES] "
+spinner &
+sleep .5
+kill "$!"
+wait $! 2>/dev/null
```

Comment:
I don't think we want to **force** a half-second delay between frames with a spinner. It does not add anything for the audience and really is just a visual distraction. Also not sure why the indent decreases here.